src/Collection.php

```diff
@@ -542,6 +586,55 @@ public function replaceDocument($documentId, array $content, array $options = []
     }
 
     /**
+     * Scrolls through specifications using the provided scrollId
+     *
+     * @param string $scrollId
+     * @param array $options Optional parameters
+     * @return mixed
+     * @throws \Exception
+     */
+    public function scrollSpecifications($scrollId, array $options = [])
```

Comment:
As we have `RolesSearchResult`, `UsersSearchResult`, we can create `SpecificationSearchResult`